README.md

```diff
@@ -1 +1,70 @@
-# oxygen
\ No newline at end of file
+# Oxygen RISC-V Simulator
+
+Oxygen is a RISC-V assembly simulator built with a Django backend. It supports the I, M, F, and C extensions, allowing users to write, decode, and execute RISC-V assembly programs. The simulator provides both full program execution and step-by-step execution modes, with features to display and update register and memory values accordingly.
+
+## Features
+
+- Write RISC-V assembly code
+- Support for I, M, F, and C extensions
+- Instruction decoding
+- Hexadecimal representation of each instruction
+- Full program processing in one go
+- Step-by-step execution
+- Display and update register and memory values during execution
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have the following installed on your system:
+
+- Python 3.x
```

Comment:
Delete these two lines

Reply:
The python and git lines have not been deleted yet. Only display the installation command